### DISCUSSION TITLE
Scroll bar design for similar style in diffrent browser

### DISCUSSION THREAD

**Discussion by knightvibhi** [heroui-inc/heroui#4687] [Feature requests]
### Is your feature request related to a problem? Please describe.

Yes, Design should look somewhat similar and should not change based on browser you use

### Describe the solution you'd like

Add `max-h-[420px]` to big dataset in table

```
 <Table
          className="px-6 min-h-[420px] max-h-[420px] overflow-y-scroll overflow-x-hidden"
          removeWrapper
          layout="fixed"
          isHeaderSticky={true}
          color="primary"
          selectionMode="single"
        >
```

### Describe alternatives you've considered

```
/* For Firefox */
/* Width */
* {
  scrollbar-width: thin;
  scrollbar-color: rgba(155, 155, 155, 0.5) rgba(0, 0, 0, 0.1);
}

/* For Webkit-based browsers */
/* Width */
*::-webkit-scrollbar {
  width: 10px;
}

/* Track */
*::-webkit-scrollbar-track {
  background: rgba(0, 0, 0, 0.1);
}

/* Handle */
*::-webkit-scrollbar-thumb {
  background-color: rgba(155, 155, 155, 0.5);
  border-radius: 5px;
}
```

Tried this css created by chatgpt

![Screenshot from 2023-11-25 15-57-11](https://github.com/nextui-org/nextui/assets/121423579/3d49f4da-427a-48e3-9066-a57699b05fff)
![Screenshot from 2023-11-25 15-57-03](https://github.com/nextui-org/nextui/assets/121423579/3e31cf77-5218-4876-bb54-063bb95942fe)

### Screenshots or Videos

![Screenshot from 2023-11-25 15-51-56](https://github.com/nextui-org/nextui/assets/121423579/912bf88e-4bc2-4720-80cd-4742b24104b1)
![Screenshot from 2023-11-25 15-51-36](https://github.com/nextui-org/nextui/assets/121423579/139dfa25-fee6-4b56-a9f9-5bf3ea3df6b4)

Here Scrollbar in firefox and chrome, both look very different and this should be fix to match the current style of ui.

**Comment by MinChanSike** [comment]
How to apply custom scrollbar component in sticky header table? I have tried but only default scrollbar is showing.
For example to wrap with [overlayscrollbars-react](https://www.npmjs.com/package/overlayscrollbars-react) or [@radix-ui/react-scroll-area](https://www.npmjs.com/package/@radix-ui/react-scroll-area)

**Comment by KhalidHassan218** [comment]
any solutions ?
